test/integ/routers/alpha-router/alpha-router.integration.test.ts

```diff
@@ -2782,9 +2910,9 @@ describe('quote for other networks', () => {
                     .equalTo(swap.estimatedGasUsedQuoteToken)
                 );
 
-                // Expect tenderly simulation to be successful
-                expect(swap.simulationStatus).toEqual(
-                  SimulationStatus.Succeeded
+                // Expect Eth Estimate Gas to not fail
+                expect(swap.simulationStatus).not.toEqual(
+                  SimulationStatus.Failed
```

Comment:
Why are we not checking success anymore? These are the simulation tests, so they must succeed.